website/content/partials/storage-entry-size.mdx

```diff
@@ -3,9 +3,17 @@ by that backend.
 
 The default entry size limit for the integrated storage backend, is 1 MiB. You
 can configure the allowable entry size with the `max_entry_size` parameter in
-your the [storage stanza](/vault/docs/configuration/storage/raft#max_entry_size).
-Vault automatically chunks any storage entry that is larger than 512 KiB but
-smaller than `max_entry_size` into smaller pieces before writing the entry to Raft.
+your the [storage
+stanza](/vault/docs/configuration/storage/raft#max_entry_size). Vault
+automatically chunks any storage entry that is larger than 512 KiB but smaller
+than `max_entry_size` into smaller pieces before writing the entry to Raft.
+
+Vault Enterprise 1.17 and higher also exposes a
+`max_mount_and_namespace_table_entry_size` configuration that allows the limit
+to be increased but only for the KV entries necessary for the mount table and
+namespace metadata. To allow for more mounts or namespaces than the default
+entry size can support, it is recommended over `max_entry_size` where available
+as it avoids unintentionally allowing other storage entries to grow very large.
```

Comment:
```suggestion
Vault Enterprise 1.17 and higher also exposes a
`max_mount_and_namespace_table_entry_size` configuration that can increase the
size limit for KV entries storing metadata related mount tables and
namespaces. We recommend increasing `max_mount_and_namespace_table_entry_size`
over `max_entry_size` when possible to avoid unintentionally allowing other
storage entries to grow very large.
```

Style correction: write in active voice